code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -701,15 +709,34 @@ pub mod pallet {
 
 					let now_seconds = T::UnixTime::now().as_secs();
 
+					let existential_deposit = T::ExistentialDeposits::get(&pool_asset);
+
+					println!("existential_deposit, {:?}", existential_deposit);
```

Comment:
no sure, may avoid having print in this code?

Reply:
yes, I was just debugging something but no print will get merged